dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils1_1.java

```diff
@@ -0,0 +1,342 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.logging.collector;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.ArrayList;
+
+import com.ibm.websphere.ras.DataFormatHelper;
+import com.ibm.ws.health.center.data.HCGCData;
+import com.ibm.ws.logging.data.GenericData;
+import com.ibm.ws.logging.data.KeyValuePair;
+import com.ibm.ws.logging.data.Pair;
+
+/**
+ *
+ */
+public class CollectorJsonUtils1_1 {
```

Comment:
please create follow on issue to make methods type-safe below (change Object event to GenericData event and remove explicit casts to GenericData) once everything is using GenericData events.  Also do for CollectorJsonUtils.